topup/types/genesis.go

```diff
@@ -2,19 +2,21 @@ package types
 
 // GenesisState is the bank state that must be provided at genesis.
 type GenesisState struct {
-	// TopupSequences map[uint256]bool `json:"topup_sequence" yaml:"topup_sequence"`
+	TopupSequences []*uint64 `json:"tx_sequences" yaml:"tx_sequences"`
```

Comment:
If you are using a pointer, you need to check if the values in the array are not nil while importing genesis. There is a method called `ValidateGenesis`. Please use that for validation.